project/src/main/webapp/common.js

```diff
@@ -0,0 +1,38 @@
+//Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// use modern JavaScript (ES5)
+"use strict"
+
+function addCommonElements() {
+  addNavbar();
+  addHeader();
+}
+
+function addNavbar() {
+  let navbar = `<a href="/index.html">Home</a>
+      <br>
+      <a href="/">Resources</a>
+      <br>
+      <a href="/">Polling Locations</a>
+      <br>`;
+  let navbarDiv = document.getElementById('navbar-container');
+  navbarDiv.innerHTML = navbar;
+}
+
+function addHeader() {
+  let header = `<h1>Voter Central</h1>`;
```

Comment:
Same comments as navbar above.
Additionally, if header will be just one HTML element, we can put it directly in the HTML.

Reply:
I was planning on possibly adding some links to it later, so I separated it out as well just in case.

Reply:
Got it, sounds great!